### PR TITLE
Add installation notes for Linux

### PR DESCRIPTION
**Pull request by gonzalo-bulnes**
## Overview

Adds some installation notes to the README in order to make the installation easier on Debian, and potentially other Linux distributions.
I didn't come up with the solutions, they were provided by @haslinghuis in #2129 and #2137. Thank you!! :slightly_smiling_face: 

## Test plan

I've verified these commands in **Debian 11 (Bullseye)** and **Betaflight Configurator 10.7.2**. As a test plan, I'd suggest:

- [x] Confirm that creating the directory as described in the README won't cause any loss of data if the directory already exists.
- [x] On Ubuntu or Debian confirm that once the directory is present, the installation succeeds without errors:
     ```
     sudo apt install ./betaflight-configurator_10.7.2_amd64.deb
     ```
- [x] Confirm that `libatomic1` can be installed on Ubuntu or Debian as described in the README.
- [x] On Fedora or another RPM-based distribution, confirm that installing the RPM package causes `libatomic` to be installed automatically:
    ```
    sudo dnf install ./betaflight-configurator-10.7.2-1.x86_64.rpm
    ```
- [x] Confirm that installing that package allows Betaflight Configurator to start.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: README.md
Comment (haslinghuis) [inline README.md:47]:
```suggestion
Post-installation errors can be prevented by making sure the directory `/usr/share/desktop-directories` exists. To make sure it exists, run the following command before installing Betaflight Configurator:
```

We prefer a more generic description, as this is true for other Linux flavors.

Comment (gonzalo-bulnes) [reply README.md:47]: I'll fix this :+1:

File: README.md
Comment (gonzalo-bulnes) [reply README.md:53]: Just confirming that I just tried this in **Fedora 34** and the package name is different (`libatomic`) **but** the RPM package specifies the dependency so it is installed automatically when installing Betaflight Configurator.

I'll apply the suggested change :+1:

Comment (gonzalo-bulnes) [reply README.md:53]: Wait, the package name is used in the first sentence. What do you think of this @haslinghuis?


> The `libatomic1` package must also be installed before installing Betaflight Configurator from the `.deb` package. (If it is missing, the installation will succeed but Betaflight Configurator will not start.) On Debian or Ubuntu you can install it as follows:

Comment (gonzalo-bulnes) [reply README.md:53]: Another option:

> The `libatomic` library must also be installed before installing Betaflight Configurator. (If it is missing, the installation will succeed but Betaflight Configurator will not start.) Some Linux distributions (e.g. Fedora) will install it automatically. On Debian or Ubuntu you can install it as follows:

Comment (gonzalo-bulnes) [reply README.md:53]: I'll apply the second option while waiting for your feedback @haslinghuis. (I also updated the test plan in the PR description accordingly.)

Comment (haslinghuis) [reply README.md]: Agreed

File: README.md
Comment (gonzalo-bulnes) [reply README.md]: Done :white_check_mark:

File: README.md
Comment (gonzalo-bulnes) [reply README.md]: Done :white_check_mark:

**Review by haslinghuis — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by Asizon — APPROVED** [review]

**Pull request merged**
